4.analyze-components/scripts/util.R

```diff
@@ -120,6 +120,13 @@ compile_reconstruction_data <- function(dataset_name, data_focus = "all") {
     reconstruction_cost_df$algorithm <-
       factor(reconstruction_cost_df$algorithm,
              levels = c("pca", "ica", "nmf", "dae", "vae"))
+    
+    reconstruction_cost_df$algorithm <- 
+      reconstruction_cost_df$algorithm %>% dplyr::recode_factor("pca" = "PCA",
```

Comment:
Why are you doing this mapping here?

Reply:
It is for the figures - I wanted to algorithms to be capitalized